aws/assume-role/mint-leaf.yml

```diff
@@ -4,54 +4,87 @@ description: Assume an AWS role
 
 parameters:
   oidc-token:
-    description: 'The OIDC token that will be exchanged for temporary credentials (e.g. vaults.your-vault.oidc.your-token)'
-    required: true
+    description: "The OIDC token that will be exchanged for temporary credentials (e.g. vaults.your-vault.oidc.your-token). Required when not using role chaining."
+    required: false
   region:
-    description: 'The AWS region (e.g. us-east-2)'
+    description: "The AWS region (e.g. us-east-2)"
     required: true
   role-to-assume:
-    description: 'The ARN of the AWS role to be assumed (e.g. arn:aws:iam::your-account-id:role/your-role)'
+    description: "The ARN of the AWS role to be assumed (e.g. arn:aws:iam::your-account-id:role/your-role)"
     required: true
   role-duration-seconds:
-    description: 'The role duration in seconds'
+    description: "The role duration in seconds"
     default: 900
   role-session-name:
-    description: 'The name of the session'
+    description: "The name of the session"
     required: false
   profile-name:
-    description: 'The profile under which the credentials will be configured'
-    default: 'default'
+    description: "The profile under which the credentials will be configured"
+    default: "default"
+  source-profile-name:
```

Comment:
Curious if anyone has a better name for this 😅 Maybe I should also make `target-profile-name` to accompany it?

Reply:
`source-profile-name-for-chaining`? 😬

Reply:
It has a description tommy 😛